platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/api/internal/DefaultLoggingSystem.java

```diff
@@ -121,13 +115,21 @@ private static Configuration createConfiguration() {
     }
 
     /**
-     * Returns the singleton instance.
+     * Returns the singleton instance. Initializes it if it wasn't initialized already
      *
      * @return The singleton instance.
      */
     @NonNull
     public static DefaultLoggingSystem getInstance() {
-        return InstanceHolder.INSTANCE;
+        // Double-checked locking for thread safety
+        if (instance == null) {
```

Comment:
While I'm fine with this and many other changes in the PR I would say that mixing that with the test changes makes the PR too complex to review. Let's split it up in 2 please

Reply:
This change, in particular, can be rollbacked. It was trying to determine why the default logger was logging initialization errors for some tests even when we instantiated a custom logger for the test. 
The reason was that whenever a `loggingMirror` was instantiated, it automatically added itself as a handler in the default logger, which, therefore, was initialized without need and without having log.properties configured, producing some logs to the emergency logger.
So in a way, there are changes in both the API and the test that solve the issue of reducing the amount of console statements in the tests.